### PR TITLE
Feat/5098  add new users to spaces easily

### PR DESCRIPTION
**Pull request by gbonaspetti**
#5098

Why this PR is on draft: missing code cleaning and clarity

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

~~- [ ] Manual, quality tests have been done~~ Tests will be made during integration

### CODE REVIEW

File: frontend/i18next.scanner/fr/translation.json [hunk 1]
```diff
@@ -24,10 +26,12 @@
   "Loading…": "Chargement…",
   "Email:": "Email :",
   "Email already exists": "Email déjà utilisé",
+  "Email notif": "Notif par mail",
```
[the remark this answers is not in the document]
Comment (lebouquetin) [reply frontend/i18next.scanner/fr/translation.json:29]: When using an abbreviation, I think the rule is to put a dot.

- `Email notif` -> `Email notif.`
- `Notif par mail` -> `Notif. par email`

[the remark this answers is not in the document]
Comment (lebouquetin) [reply frontend/i18next.scanner/fr/translation.json:29]: @raphj @Skylsmoi you confirm?

[the remark this answers is not in the document]
Comment (raphj) [reply frontend/i18next.scanner/fr/translation.json:29]: it should be okay for English, notif is referenced in the Wiktionary without indication it is rare or misspelled:  https://en.wiktionary.org/wiki/notif

[the remark this answers is not in the document]
Comment (raphj) [reply frontend/i18next.scanner/fr/translation.json:29]: marked as slang though.

[the remark this answers is not in the document]
Comment (gbonaspetti) [reply frontend/i18next.scanner/fr/translation.json:29]: The "mail" is a typo, I'm gonna fix and add the dots

**Review by Haskaris — APPROVED** [review]

**Pull request merged**